src/test/java/com/tjtanjin/steve/ui/UiHandlerTest.java

```diff
@@ -43,15 +43,15 @@ void showWelcome_whenInvokePrintln_thenOutputCaptorSuccess() {
     @Test
     void showInfo_whenInvokePrintln_thenOutputCaptorSuccess() {
         String expected = "Test print information message!";
-        uiHandler.showInfo(expected);
+        uiHandler.showResponse(expected);
         assertEquals(expected, outputStreamCaptor.toString().trim());
     }
 
     @Test
     void showError_whenInvokePrintln_thenOutputCaptorSuccess() {
         String errMsg = "Test print error message!";
         String expected = "Error: " + "Test print error message!";
```

Comment:
Would it be better to name this expectedMsg?

Reply:
Hello! Thank you for the input! I agree that the variable name could be made clearer :D Fixed in the latest commit!